crates/swc_ecma_minifier/tests/fixture/next/wrap-contracts/output.js

```diff
@@ -25973,7 +25973,8 @@ class Zip {
                                 throw Error(`Support for ${contractDefinition.srcWasmLang} not implemented yet.`);
                         }
                         return this.logger.info(`WASM ${contractDefinition.srcWasmLang} handler created in ${benchmark.elapsed()}`), new WasmHandlerApi_1.WasmHandlerApi(swGlobal, contractDefinition, jsExports || wasmInstance.exports);
-                    } else {
+                    }
+                    {
```

Comment:
@Austaras @kdy1 This PR drops the `else` statement here. Is this expected? Thanks~

Reply:
It is. The if branch before it has a return in the end so this else is unnecessary.

Reply:
I see, thanks for clarifying!